dev/docker-compose.yaml

```diff
@@ -233,8 +233,11 @@ services:
       REQUEST_CACHE_URL: redis://redis:6379/1
       SESSION_REDIS: redis://redis:6379/0
       PREFERRED_URL_SCHEME: https
-      FHIR_URL:  "http://fhir-internal:8080"
+      FHIR_URL:  "http://fhir-internal:8080/fhir/"
       LOGSERVER_URL: "https://logs.${BASE_DOMAIN:-localtest.me}"
+      TWILIO_WEBHOOK_CALLBACK: "https://messagingservice.${BASE_DOMAIN:-localtest.me}/sms"
```

Comment:
I believe this should just be `TWILIO_WEBHOOK_CALLBACK: "https://messagingservice.${BASE_DOMAIN:-localtest.me}"`

Reply:
ah, I see.  you must register the `/sms` view directly in Twilio's web UI.